apps/gallery/style/gallery.css

```diff
@@ -41,6 +41,10 @@ section {
   display: none;
 }
 
+body.showing-dialog > section {
```

Comment:
To be honest, I don't actually understand why this is necessary. Is the screen reader not able to tell that these elements are obscured by the opaque dialog? Is that a bug in the screen reader?  Is it normal to have to do something like this for accessible dialogs, or is there something weird about the way that we do dialogs in this app that is forcing us to do it this way?

If you could explain these issues in a comment here in the gallery.css file, that would be really nice.